main.go

```diff
@@ -19,6 +19,15 @@ func main() {
 	cl := oauth2.NewClient(oauth2.NoContext, ts)
 	ghClient := github.NewClient(cl)
 
+	// set up staging dir
+	stagingDirName := "staging"
+	if _, err := os.Stat(stagingDirName); os.IsNotExist(err) {
+		err := os.Mkdir(stagingDirName, os.ModePerm)
+		if err != nil {
+			log.Fatalf("Error creating dir %s (%s)", stagingDirName, err)
+		}
+	}
```

Comment:
does the staging dir need to be created on *all* runs of the CLI? seems like only needs to be created on a `helm-chart *` command